### PR TITLE
Navbar styling

### LINKED ISSUE #25: Standardize Navbar across Real Dev Squad
### Design of New Navbar:
https://www.figma.com/file/bpFHG9hXM13GSLmTkoNsrF/Navbar?node-id=0%3A1
### For CSS, refer to the link below
https://github.com/Real-Dev-Squad/website-www/blob/develop/css/navbar.css
## Few Details

- font : Roboto
- weight: 700
- active color : #49A82E
- hover color : #49A82E
- font size : 16 px (try making it in rem)
- active underline height : 3px (color : #49A82E)
-
### Pay Attention
=> Underline should be under the text as shown in the image and not inside the container in which the text persists.
=> Underline should be only under the active tab, it should not be visible on hover.
=> The full navbar should be changed to the toggle menu when the screen width is less than 970px.
=> User should be directed to my.realdevsquad.com on clicking User Greet

### There might be slight changes for different domains and you are free to ask in the comments for that

Refer to https://github.com/Real-Dev-Squad/website-members/pull/265 for more insights.

### PR DESCRIPTION
**Pull request by pallabez**
### Issue
Closes #25

#### Before
image
image

#### After
image
image

Also, fixed navbar to top and added animation. 

### Is Development Tested?

- [x] Yes
- [ ] No

### CODE REVIEW

File: components/Navbar.vue [hunk 1]
```diff
@@ -1,43 +1,50 @@
...
+    </ul>
+
+    <ul class="dropdown" :class="{ 'dropdown--open': isClicked }">
+      <li>
+        <a href="https://realdevsquad.com/" class="dropdown__option">Home</a>
+      </li>
+      <li v-for="item in LINKS" :key="item.name">
+        <a :href="item.link" class="dropdown__option">{{ item.name }}</a>
+      </li>
+    </ul>
```
Comment (pallabez) [inline components/Navbar.vue:32]: Do let me know if there is a way to merge this two into one.

File: components/Navbar.vue [hunk 2]
```diff
@@ -1,43 +1,50 @@
...
-          <a class="active" href="https://goals.realdevsquad.com/">Goals</a>
-        </li>
-      </ul>
-    </nav>
-  </header>
+  <nav class="navbar">
+    <button class="hamburger" @click="toggleClicked(isClicked)">
+      <div class="hamburger__bar" />
+      <div class="hamburger__bar" />
+      <div class="hamburger__bar" />
```
Comment (rohan09-raj) [inline components/Navbar.vue:6]: We can use the here maybe or if we are not using the `menu.png`, can we remove it ?

File: components/Navbar.vue (hunk 1 above)
Comment (rohan09-raj) [reply components/Navbar.vue:32]: Yes we can merge this by using two separate elements for `Home` as we are using here and set the `display: none` and `display: block` properties as per the screen sizes using the media query
And for the rest of the links we can use simple CSS for the appearance with media query

Comment (rohan09-raj) [reply components/Navbar.vue:32]: You can refer to this PR or else we can connect on Discord
https://github.com/Real-Dev-Squad/website-status/pull/264/files

File: components/Navbar.vue [hunk 3]
```diff
@@ -1,43 +1,50 @@
...
+      <li v-for="item in LINKS" :key="item.name">
+        <a :href="item.link" class="dropdown__option">{{ item.name }}</a>
+      </li>
+    </ul>
+
+    <button
+      class="login"
+      onClick="window.location.href='https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97'"
+    >
+      <span class="login__text">Sign In With GitHub</span>
```
Comment (rohan09-raj) [inline components/Navbar.vue:38]: How you are changing the text for different screen sizes i.e. `Sign In` for less than 768px and `Sign In With GitHub` for more than 768px ??

File: components/Navbar.vue [hunk 4]
```diff
@@ -57,85 +71,98 @@ export default {
...
-  width: 35px;
-  height: 70%;
-  left: 10px;
-  flex-flow: column;
-  justify-content: space-around;
   align-items: center;
-  padding: 10px 0;
-  box-sizing: border-box;
-  cursor: pointer;
+  background-color: #1d1283;
```
Comment (rohan09-raj) [inline components/Navbar.vue:78]: Can we please have constants for the colors used across the CSS ?

File: components/Navbar.vue [hunk 5]
```diff
@@ -57,85 +71,98 @@ export default {
...
 }
-.hamburger-menu span {
-  background-color: white;
-  width: 70%;
+.hamburger__bar {
   height: 3px;
-  margin: 2px 0;
+  width: 25px;
+  margin: 5px 0;
+  background-color: white;
```
Comment (rohan09-raj) [inline components/Navbar.vue:96]: Can we please have constants for the colors used across the CSS and keep it consistent with Hex?

File: components/Navbar.vue [hunk 6]
```diff
@@ -57,85 +71,98 @@ export default {
...
+.redirection__option {
+  padding: 19px 16px;
+  margin: 0 10px;
+  border-radius: 10px;
+}
+.redirection__option--logo {
+  margin-right: 20px;
+}
+.redirection__option:hover {
+  color: #49a82e;
```
Comment (rohan09-raj) [inline components/Navbar.vue:111]: Can we please have constants for the colors used across the CSS and keep it consistent with Hex?

File: components/Navbar.vue [hunk 7]
```diff
@@ -57,85 +71,98 @@ export default {
...
-  text-align: center;
-  font-weight: bolder;
-  padding: 0 0.5em;
-  text-decoration: none;
 }
-nav li a:hover,
-nav li a.active {
-  color: #87d870;
+.dropdown__option:hover {
+  color: #49a82e;
```
Comment (rohan09-raj) [inline components/Navbar.vue:134]: Can we please have constants for the colors used across the CSS and keep it consistent with Hex?

File: components/Navbar.vue [hunk 8]
```diff
@@ -57,85 +71,98 @@ export default {
...
+}
+.login__text {
+  overflow: hidden;
+  white-space: nowrap;
 }
-img {
-  width: 3.5em;
-  height: 3.5em;
+.login__image {
+  margin-left: 0.3rem;
```
Comment (rohan09-raj) [inline components/Navbar.vue:151]: Can we please be consistent with the units i.e. px for margin and padding

File: components/Navbar.vue (hunk 3 above)
Comment (pallabez) [reply components/Navbar.vue:38]: I made overflow hidden and decreased the length of the button when screen reaches that breakpoint.

File: components/Navbar.vue (hunk 1 above)
Comment (pallabez) [reply components/Navbar.vue:32]: Gotcha :+1:

File: components/Navbar.vue (hunk 2 above)
Comment (pallabez) [reply components/Navbar.vue:6]: The png isn't that high quality, that's why avoided it.

Comment (rohan09-raj) [reply components/Navbar.vue:6]: Got it

File: components/Navbar.vue (hunk 3 above)
Comment (rohan09-raj) [reply components/Navbar.vue:38]: Oh, nice, but will the size of the button change with screen sizes or is it fixed with `px` ?

Comment (pallabez) [reply components/Navbar.vue:38]: It is fixed

**Comment by pallabez** [comment]
Need to add goals in navigation as well.

File: components/Navbar.vue [hunk 9]
```diff
@@ -1,43 +1,50 @@
...
+        <a href="https://realdevsquad.com/" class="dropdown__option">Home</a>
+      </li>
+      <li v-for="item in LINKS" :key="item.name">
+        <a :href="item.link" class="dropdown__option">{{ item.name }}</a>
+      </li>
+    </ul>
+
+    <button
+      class="login"
+      onClick="window.location.href='https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97'"
```
Comment (rohan09-raj) [inline components/Navbar.vue:36]: Can we please put this link in a constant ?

**Pull request merged**